### ISSUE TITLE
[1.0.4] Metal Former Refine mode Issues

### ISSUE THREAD

**Issue by Renniuq** [LambdaInnovation/AcademyCraft#494]
So, in Refine mode, its supposed to double ores. For one, Imag Silicon ore is instead quadrupled, though that might be intentional. Also, it seems to prefer IC2 outputs, choosing that form of tin, lead, and copper when possible. However, it doesn't process any other ores that aren't in AcademyCraft, IC2, or vanilla. It flat out refuses to accept ProjectRed's copper and tin ores, and yet will take any other ore (from what I can tell) into its input, but will not process them. This applies to Immersive Engineering's ores, Thermal Foundation, and Tinker's Construct. However, it will take Forestry's copper and tin and process that just fine. I'm not sure how its deciding what can be put into the first slot and what can be processed by the machine. I'm guessing one of the most universally applicable ways would be something along the lines of what Another One Bites the Dust does, namely taking the oredictionary and using entries with ingot and ore and adding dust/molten/crushed/etc versions so they work with all ore processing types.

**Comment by Wolfseverest93** [comment]
That's because the dev for ProjectRed hasn't properly connected the ores in the oredictionary, as this block runs based on an oredict recognization. The only reason it favors IC2 when it's in the pack is because IC2 is always the first to register into the oredictionary files. As for thermal foundation, immersive engineering, and tinkers construct, are you sure you're using the latest versions? They work perfectly fine for me on the latest 1.7.10 versions so far. I can also verify that it also works properly with processing terrafirmacraft ores (namely the ores related to redstone output, lapiz output, and coal output), just to get that out of the way. If you have any mods that alter recipe processing (similar to how nomorerecipeconflict allows swapping between recipe outputs for conflicting recipes) it could have the potential to have the effect you speak of in many cases, the exception being minetweaker 3, as that requires user interference to do any modifications at all. If your problem persists, I would suggest just using the tinkers construct smeltery, or one of the production increasing methods available in IC2, TE, or IE until your issue can be replicated and resolved. If a dev would, using a debug built version of the mod that logs to file each action of the metal former would also help pinpoint the possible cause and resolution. @Renniuq you mind passing me your mod list that you have this issue with? I'll run some tests on my spare time.

**Comment by WeAthFoLD** [comment]
One word about issue submission: Please **describe your problem briefly and clearly**. It is best if one can identify the core problem of the issue by a mere look, not spending time looking through a whole long paragraph trying to figure out what you mean.

Please give a **TLDR**.

**Comment by ShieLian** [comment]
I wrote the recipes with OreDict, but i forget the details.I'll check the implement if I'm free.

**Comment by Wolfseverest93** [comment]
My post wasn't an issue, it was just me offering to help in what ways I can to resolve the issue @Renniuq seems to be having.

Btw, I see you've made really exception progress with the mod in the days I've been working @WeAthFoLD &crew. Great job.

**Comment by Renniuq** [comment]
I have since stopped using the pack where this was occurring, it was a custom one for a small server for some friends, and has since been changed. However, it does seem, at least with the forestry ores, the issue occurs in the most recent version of Regrowth (v1.0.1). That being said, we have Mekanism there, and we've been using that for ore processing mainly, so the issue is not very pressing atm.

I don't think this needs a tl;dr, does it?

here's one anyways: in Regrowth v1.0.1 forestry ores still don't process.
